Muvr/MRAppDelegate.swift

```diff
@@ -59,7 +59,7 @@ protocol MRApp : MKExercisePropertySource {
     /// - parameter start: the start date, typically value ``NSDate()``
     /// - parameter id: the session identity, typically ``NSUUID().UUIDString``
     ///
-    func startSessionForExerciseType(exerciseType: MKExerciseType, start: NSDate, id: String) throws
+    func startSession(forExerciseType exerciseType: MKExerciseType) throws
```

Comment:
Can we have just one startSession function? This bit in MRAppDelegate is scary (particularly without comments!)

```
    func sessionClassifierDidStart(session: MKExerciseSession) {
        try! startSessionForExerciseType(session.exerciseType, start: session.start, id: session.id)
    }
    
    func startSession(forExerciseType exerciseType: MKExerciseType) throws {
        try startSessionForExerciseType(exerciseType, start: NSDate(), id: NSUUID().UUIDString)
        connectivity.startSession(MKExerciseSession(managedSession: currentSession!))
    }
    
    func startSessionForExerciseType(exerciseType: MKExerciseType, start: NSDate, id: String) throws {
        ...
   }
```